src/kiota/Handlers/Plugin/AddHandler.cs

```diff
@@ -43,6 +53,8 @@ public override async Task<int> InvokeAsync(InvocationContext context)
     {
         string output = context.ParseResult.GetValueForOption(OutputOption) ?? string.Empty;
         List<PluginType> pluginTypes = context.ParseResult.GetValueForOption(PluginTypesOption) ?? [];
+        PluginAuthType pluginAuthType = context.ParseResult.GetValueForOption(PluginAuthTypeOption);
```

Comment:
Just checking here. Should these be used to set the configuration object? Or is that to be handled in a separate PR?

Reply:
see https://github.com/microsoft/kiota/pull/5209

Reply:
Updated